bin/apisix

```diff
@@ -113,9 +113,8 @@ env APISIX_PROFILE;
 
 {% if stream_proxy then %}
 stream {
-    lua_package_path  "$prefix/deps/share/lua/5.1/?.lua;/usr/share/lua/5.1/apisix/lua/?.lua;]=]
-                      .. [=[/usr/local/share/lua/5.1/apisix/lua/?.lua;]=]
-                      .. [=[$prefix/deps/share/lua/5.1/apisix/lua/?.lua;]=]
+    lua_package_path  "$prefix/deps/share/lua/5.1/?.lua;]=]
+                      .. [=[$prefix/deps/share/lua/5.1/apisix/?.lua;]=]
```

Comment:
I think we can delete this line

Reply:
which line?

Reply:
`$prefix/deps/share/lua/5.1/apisix/?.lua;`

Reply:
line 61 `pkg_path  = apisix_home .. "/lua/?.lua;"` also need to change to `pkg_path  = apisix_home .. "/?.lua;"`